### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by romankuchera**

### CODE REVIEW

File: _includes/about/about.scss [hunk 1]
```diff
@@ -6,12 +6,17 @@
...
+      padding: 0 10px;
+    }
+
     @include tablet-screen {
       @include flex(column, center, center);
       padding: 0 100px;
     }
 
     @include small-tablet-screen {
+      max-width: 460px !important;
```
Comment (nazarimplex) [inline _includes/about/about.scss:19]: Can we handle it without `!important`?

File: _includes/advantages/advantages.scss [hunk 2]
```diff
@@ -51,11 +51,20 @@
     .advantage {
       display: grid;
       grid-template-columns: 66px 294px;
```
Comment (nazarimplex) [inline _includes/advantages/advantages.scss:53]: Do we need to hardcode width of second column at all? 
Why isn't it just 1fr? It should take the rest of the space in the row.
And you will not need so many media-queries to update that hardcoded width

File: _includes/how-it-works/how-it-works.scss [hunk 3]
```diff
@@ -3,10 +3,14 @@
 .how-it-works-section {
   background-color: white;
 
+  @include small-tablet-screen {
+    padding: 122px 0 !important;
```
Comment (nazarimplex) [inline _includes/how-it-works/how-it-works.scss:7]: Why `!important`?

**Review by nazarimplex — CHANGES_REQUESTED** [review]

File: _includes/how-it-works/how-it-works.scss (hunk 3 above)
Comment (romankuchera) [reply _includes/how-it-works/how-it-works.scss:7]: because we need to rewrite styles from style.scss

File: _includes/about/about.scss (hunk 1 above)
Comment (romankuchera) [reply _includes/about/about.scss:19]: no, because we need to rewrite styles from style.scss

**Review by nazarimplex — APPROVED** [review]

**Pull request merged**